homeassistant/components/drop_connect/entity.py

```diff
@@ -0,0 +1,52 @@
+"""Base entity class for DROP entities."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    CONF_DEVICE_DESC,
+    CONF_DEVICE_NAME,
+    CONF_DEVICE_OWNER_ID,
+    CONF_DEVICE_TYPE,
+    CONF_HUB_ID,
+    DEV_HUB,
+    DOMAIN,
+)
+from .coordinator import DROPDeviceDataUpdateCoordinator
+
+
+class DROPEntity(CoordinatorEntity[DROPDeviceDataUpdateCoordinator]):
+    """Representation of a DROP device entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, entity_type: str, coordinator: DROPDeviceDataUpdateCoordinator
+    ) -> None:
+        """Init DROP entity."""
+        super().__init__(coordinator)
+        if TYPE_CHECKING:
+            assert coordinator.config_entry is not None
```

Comment:
```suggestion
```

Reply:
Removed in ff80af4b820b7e89659c2a597bdc86ab1f28fd5d.